src/structures/ApplicationServer.ts

```diff
@@ -37,7 +44,7 @@ export class ApplicationServer {
     public description: string | undefined;
 
     /** The current processing status of the server. */
-    public status: string | null;
+    public status: ApplicationServerStatus | null;
```

Comment:
Ideally this should use the `ServerStatus` enum but there is a possibility that this could be a breaking change.